regcore/db/django_models.py

```diff
@@ -40,9 +40,16 @@ def _transform(self, reg, version):
 
     def bulk_put(self, regs, version, root_label):
         """Store all reg objects"""
-        # This does not handle subparts. Ignoring that for now
-        Regulation.objects.filter(version=version,
-                                  label_string__startswith=root_label).delete()
+        # Delete any existing regulation objects for this version and
+        # root label.
+        # XXX: This does not handle subparts. Ignoring that for now
```

Comment:
Right off, I'm not sure. That comment seems to come from here: https://github.com/willbarton/regulations-core/commit/e7d7afb07ea95ad751b25bec89563a537ced6483